src/main/kotlin/edu/carroll/polyominoes/web/rest/ajax/GameStats.kt

```diff
@@ -0,0 +1,48 @@
+package edu.carroll.polyominoes.web.rest.ajax
+
+import jakarta.validation.constraints.NotNull
+import jakarta.validation.constraints.Size
+import java.time.Duration
+
+/**
+ * Represent the statistics from the game send via ajax.
+ */
+class GameStats() {
+
+    constructor(time: Double, level: Long, score: Long, rows: Long, pieces: Long) : this() {
+        this.time = Duration.ofMillis((time * 1000).toLong())
```

Comment:
Mixing data types (Double and Duration) is just asking for trouble.  Either use Duration everywhere, or some other data type that doesn't require converting the data to/from the different data types.